src/module.rs

```diff
@@ -61,14 +56,24 @@ pub fn init(_module: ModuleContext) -> bool {
         service_instance, skywalking_version, "Starting skywalking agent"
     );
 
+    let worker_addr = {
+        match tempfile::NamedTempFile::new() {
+            Err(e) => {
+                error!("Create named temporary file failed: {}", e);
+                return true;
+            }
+            Ok(f) => f.into_temp_path().to_str().unwrap().to_string(),
```

Comment:
Should not use `unwrap()`, please use `error!()` and `return`.

Reply:
Though it should rarely fail here, I'll update it.